index.js

```diff
@@ -1,40 +1,72 @@
 const {
   fetchJSON,
   getOwner,
-  getRepo
+  getRepo,
+  getComment,
+  isBodyFollowingTemplate,
+  getFinalComment,
+  parseBody
 } = require('./util')
 /**
  * This is the entry point for your Probot App.
  * @param {import('probot').Application} app - Probot's Application class.
  */
-var base64 = require('base-64')
 module.exports = app => {
-  // Your code here
   app.log('Yay, the app was loaded!')
 
   app.on('issues.opened', async context => {
     const issueBody = (context.payload.issue.body)
-    const issueComment = context.issue({ body: 'Thanks for opening this issue!' })
+    const issueCommentOnSuccess = getComment(context, 'Thank you for opening this issue')
+    const issueCommentOnFailure = getComment(context, 'Please abide by the template ISSUE_TEMPLATE.md while opening issues')
     const owner = getOwner(context)
     const repo = getRepo(context)
     /**
-    * Fetch contents of readme.md containing
+    * Fetch contents of ISSUE_TEMPLATE.md containing
     * content in base64 format
     */
     const templateBody = await fetchJSON(
-      `https://api.github.com/repos/${owner}/${repo}/contents/README.md`
+      `https://api.github.com/repos/${owner}/${repo}/contents/ISSUE_TEMPLATE.md`
```

Comment:
Can you check [this article here](https://help.github.com/articles/manually-creating-a-single-issue-template-for-your-repository/), a user might put the `issue_template` at other places as well. for e.g. `.github/ISSUE_TEMPLATE.md` is one of the common places for storing the issue/pr template

For now, it looks good but in future you might want to work on this as well. :smiley:

Reply:
Will check this out and will send the improvement in the next PR! 👍

Reply:
cool!

Reply:
for a start, I think it’s okay to just look for `ISSUE_TEMPLATE.md`. You can implement other file names/locations in the next iterations

Reply:
Done!

Reply:
Sure, I'll add this abstraction and will handle other locations in a follow up PR. I have made the said changes to the readme.
In case everything else looks good, could you please merge it so that I may deploy it on glitch and we can all try it out?